applications/main/subghz/subghz_cli.c

```diff
@@ -23,6 +23,7 @@
     "299999755...348000000 or 386999938...464000000 or 778999847...928000000"
 
 #define SUBGHZ_REGION_FILENAME "/int/.region_data"
+#define SUBGHZ_CALIBRATION_FILENAME "/int/.subghz.calibration"
```

Comment:
We'd rather to see it on ext storage in subghz assets folder, but that will require some tests first.